bundles/org.openhab.binding.lutron/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -84,6 +84,21 @@
 				<label>Fade Out Time</label>
 				<description>Fade time in seconds when turning off the light</description>
 			</parameter>
+			<parameter name="onLevel" type="decimal" min="0.01" max="100.00">
+				<label>On Level</label>
+				<description>Output level to go to when an ON command is received. Default is 100%.</description>
+				<default>100</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="onToLast" type="boolean">
+				<label>Turn on to last level</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions